README.md

```diff
@@ -0,0 +1,29 @@
+## theme-example for ownCloud
+
+### Use Theme
+To use the theme clone or download this repository to the `owncloud\apps` folder on your server. Please note that the Apache user needs access rights to the folder and its content. To adjust the permissions use `chmod -R www-data:www-data theme-example`.
```

Comment:
Proposal:
`### Use Theme` --> `### Use the theme-example App` 
`To use the theme clone...` --> `To use the theme-example app, clone...`
`To adjust the permissions...` --> `As an example, use following command to adjust the permissions...`

Reply:
@mmattel Thanks Martin, I'll take your suggestions.